client/js/analysis.js

```diff
@@ -1,27 +1,86 @@
-
-
+// Will be overwritten in merge
 window.onload = function() {
```

Comment:
nit: prefer `addEventListener('onload', () => ...)` to `window.onload = function() { ... }`